backend/routes/eletrodomesticoRoutes.js

```diff
@@ -1,9 +1,10 @@
 import express from 'express';
 const router  = express.Router();
-import { getEletrodomesticos, getEletrodomestico, criarEletrodomestico, atualizarEletrodomestico, deleteEletrodomestico, criarEletrodomesticoReview} from '../controllers/eletrodomesticoController.js';
+import { getEletrodomesticos, getEletrodomestico, criarEletrodomestico, atualizarEletrodomestico, deleteEletrodomestico, criarEletrodomesticoReview, getTopEletrodomesticos} from '../controllers/eletrodomesticoController.js';
 import {protect, admin} from '../middleware/authMiddleware.js'
 router.get('/', getEletrodomesticos);
 router.get('/:id', getEletrodomestico);
+router.get('/top', getTopEletrodomesticos)
```

Comment:
Mover a linha para cima:


```suggestion
router.get('/top', getTopEletrodomesticos)
router.get('/:id', getEletrodomestico);
```